src/OpenLoco/src/Windows/ObjectSelectionWindow.cpp

```diff
@@ -249,28 +265,74 @@ namespace OpenLoco::Ui::Windows::ObjectSelectionWindow
         }
 
         // Add a marker to denote the last tab
-        _tabInformation[tabPos].index = 0xFF;
+        _tabPositions[tabPos].index = 0xFF;
 
-        const auto firstTabIndex = ObjectType(_tabInformation[0].index);
+        const auto firstTabIndex = ObjectType(_tabPositions[0].index);
         repositionTargetTab(self, firstTabIndex);
     }
 
-    // 0x00472BBC
-    static ObjectManager::ObjIndexPair sub_472BBC(Window* self)
+    static bool contains(const std::string_view& a, const std::string_view& b)
     {
-        const auto objects = ObjectManager::getAvailableObjects(static_cast<ObjectType>(self->currentTab));
+        return std::search(a.begin(), a.end(), b.begin(), b.end(), [](char a, char b) {
```

Comment:
should we worry about utf8?

Reply:
Eventually, yes, but currently all locale strings are converted from UTF-8 to ISO 8859-1 on read. We could probably start a refactor to switch to UTF-8 at this point, though.